### PR TITLE
feat: Add isOverlay, isReady and focused properties to WebView class

### PR DESCRIPTION
**Pull request by C0kkie**

### CODE REVIEW

File: client/index.d.ts [hunk 1]
```diff
@@ -1025,6 +1025,27 @@ declare module "alt-client" {
...
+     * @alpha
+     */
+    public readonly ready: boolean;
+
+    /**
+     * Is the webview focused.
+     *
+     * @alpha
+     */
+    public readonly focused: boolean;
```
Comment (LeonMrBonnie) [inline client/index.d.ts:1047]: Remove readonly

Comment (C0kkie) [reply client/index.d.ts]: done

File: client/index.d.ts [hunk 2]
```diff
@@ -1025,6 +1025,27 @@ declare module "alt-client" {
...
+     * @alpha
+     */
+    public readonly isOverlay: boolean;
+
+    /**
+     * Is the webview ready.
+     *
+     * @alpha
+     */
+    public readonly ready: boolean;
```
Comment (LeonMrBonnie) [inline client/index.d.ts:1040]: Change to `isReady`

Comment (C0kkie) [reply client/index.d.ts]: changed

**Pull request merged**